scripts/coreir_interface_cleaner.cpp

```diff
@@ -0,0 +1,219 @@
+// This file is a container for a bunch of coreir hacks
```

Comment:
Is this going to be part of standard coreir pass or just a hack inside Garnetflow.

Reply:
It is just going to be a hack inside garnetflow. It just changes the ROM parameterization and linebuffer / unified buffer namespaces to match what the current CGRAMapper will accept.